vendor/k8s.io/apimachinery/pkg/api/testing/roundtrip/roundtrip.go

```diff
@@ -93,9 +93,9 @@ func RoundTripProtobufTestForScheme(t *testing.T, scheme *runtime.Scheme, fuzzin
 
 var FuzzIters = flag.Int("fuzz-iters", 20, "How many fuzzing iterations to do.")
 
-// globalNonRoundTrippableTypes are kinds that are effectively reserved across all GroupVersions
+// GlobalNonRoundTrippableTypes are kinds that are effectively reserved across all GroupVersions
 // They don't roundtrip
-var globalNonRoundTrippableTypes = sets.NewString(
+var GlobalNonRoundTrippableTypes = sets.NewString(
```

Comment:
can we keep this private? As far as I see it is not used outside of the package.

Reply:
Yes, I make it available b/c we copy&pasted in origin. But not that this will be upstream I don't care anymore.

Reply:
ic